unizip-readme.txt

```diff
@@ -4,27 +4,24 @@ To run the game, extract the engine archive for your operating system
 into this subdirectory, so the files are right outside the pkg/ directory.
 Then, run the executable extracted from the archive.
 
-Engine archives are named like <system>-<architecture>.zip
-
-System is either, linux, windows, or macos.
-
-Architecture is either:
-
-- amd64: 64-bit x86 processor
-- i686: 32-bit x86 processor
-- arm64: 64-bit arm processor
-- armhf: 32-bit arm processor with hardware float
-
-For example windows-amd64.zip is the engine archive for running
-the game on the Windows operating system for 64-bit x86 computers.
+NOTE: In Windows Explorer using the "Extract all" function creates an unwanted extra directory,
```

Comment:
It's also the same on Linux with most file browser, like Nautilus (GNOME). So we may rewrite that in a more generic way.

Reply:
Done.